apps/condo/domains/news/schema/NewsItemTemplate.test.js

```diff
@@ -0,0 +1,283 @@
+/**
+ * Generated by `createschema news.NewsItemTemplate 'organization?:Relationship:Organization:CASCADE; title:Text; body:Text;'`
+ */
+
+const faker = require('faker')
+
+const {
+    makeLoggedInAdminClient,
+    makeClient,
+    UUID_RE,
+    DATETIME_RE,
+    expectToThrowGQLError,
+} = require('@open-condo/keystone/test.utils')
+const {
+    expectToThrowAuthenticationErrorToObj,
+    expectToThrowAuthenticationErrorToObjects,
+    expectToThrowAccessDeniedErrorToObj,
+    expectToThrowAccessDeniedErrorToObjects,
+} = require('@open-condo/keystone/test.utils')
+
+const {
+    NewsItemTemplate,
+    createTestNewsItemTemplate,
+    updateTestNewsItemTemplate,
+} = require('@condo/domains/news/utils/testSchema')
+const {
+    createTestOrganization,
+    makeClientWithRegisteredOrganization,
+    createTestOrganizationEmployeeRole,
+    createTestOrganizationEmployee,
+} = require('@condo/domains/organization/utils/testSchema')
+const {
+    makeClientWithNewRegisteredAndLoggedInUser,
+    makeClientWithSupportUser,
+    makeClientWithStaffUser,
+    makeClientWithResidentUser,
+} = require('@condo/domains/user/utils/testSchema')
+
+let adminClient, supportClient, userClient, residentClient, anonymousClient, staffClient, staffClientNoPermission,
+    dummyO10n, dummyNewsItemTemplate
+
+describe('NewsItemTemplate', () => {
+    beforeAll(async () => {
+        adminClient = await makeLoggedInAdminClient()
+    })
+
+    describe('CRUD tests', () => {
+        beforeAll(async () => {
+            supportClient = await makeClientWithSupportUser()
+            userClient = await makeClientWithNewRegisteredAndLoggedInUser()
+            await makeClientWithRegisteredOrganization()
+            residentClient = await makeClientWithResidentUser()
+            anonymousClient = await makeClient()
+            staffClient = await makeClientWithStaffUser()
+            staffClientNoPermission = await makeClientWithStaffUser()
+
+            const [templateCreated] = await createTestNewsItemTemplate(adminClient)
+            dummyNewsItemTemplate = templateCreated
+
+            const [o10n] = await createTestOrganization(adminClient)
```

Comment:
Strange naming for me. Does it make sense?

Reply:
It's a short name for `o r g a n i z a t i o n`, Like i18n :)